src/components/CurrentWeather/DailyWeather.jsx

```diff
@@ -1,31 +1,29 @@
 import DailyWeatherSingle from './DailyWeatherSingle';
 import { Box } from '@mui/material';
 import { useSelector } from 'react-redux';
+import { getWeatherInfo } from '../../redux/slices/weatherSlice';
 
 const DailyWeather = () => {
-    const weatherInfo = useSelector((state) => state.weatherInfoSlice.value);
+    const weatherInfo = useSelector((state) => getWeatherInfo(state));
 
     return (
         <Box
             sx={{
                 display: 'flex',
-                flexDirection: 'row',
+                flexDirection: 'column',
                 gap: 2,
-
-                overflowX: 'auto',
-                height: '320px',
-                marginLeft: 2,
                 paddingBottom: 1,
-                marginTop: {
-                    xs: 2,
-                    sm: 2,
-                    md: 2,
-                    lg: 0,
-                    xl: 0,
-                },
+                // marginTop: {
+                //     xs: 2,
+                //     sm: 2,
+                //     md: 2,
+                //     lg: 0,
+                //     xl: 0,
+                // },
             }}
+            elevation={4}
         >
-            {weatherInfo.daily.map((singleDay) => {
+            {weatherInfo.daily.slice(1).map((singleDay) => {
```

Comment:
This is super aggresive statement - what will happen if weatherInfo is null or daily is null, or daily is not an array?